_posts/weifeng0102/2021-11-17-weifeng-interface-draft-and-revised-plan.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+author: <weifeng0102>
```

Comment:
With the author defined this way, I don't think your post will show up on the People page under your name. Check to make sure your name in the `config.yml` [file](https://github.com/inf380p/inf380p.github.io/blob/production/_config.yml) and this header match up. You can refer to [this short video](https://utexas.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=09c72ee6-9baf-4cb9-b4fa-ade3012ab27f) in the Panopto folder.